rcqt.so/src/action.cpp

```diff
@@ -55,7 +55,7 @@
 #define JSON_CREATE_NODE_ERROR "JSON cannot create node"
 #define JSON_PKGCHK_NODE_NAME "pkgchk"
 #define PACKAGE "package"
-#define PACKAGELIST "packagelist"
+#define PACKAGELIST (getOS() == OSType::Ubuntu?"debpackagelist":"rpmpackagelist")
```

Comment:
everytime PACKAGELIST appears, a call will be made to getOS() because of #define .  since this is constant to a machine, use const variable initialized to this value, which is definitely cheaper than calls to getOS()